starport/pkg/cosmoscmd/proxy.go

```diff
@@ -0,0 +1,75 @@
+package cosmoscmd
+
+import (
+	"path/filepath"
+	"time"
+
+	"github.com/cosmos/cosmos-sdk/client"
+	"github.com/cosmos/cosmos-sdk/server"
+	"github.com/spf13/cobra"
+	"github.com/tendermint/starport/starport/pkg/cosmosutil"
+	"github.com/tendermint/starport/starport/pkg/gitpod"
+	"github.com/tendermint/starport/starport/pkg/xchisel"
+	"github.com/tendermint/starport/starport/services/network/networkchain"
+)
+
+const TunnelRerunDelay = 5 * time.Second
+
+func startProxyForTunneledPeers(clientCtx client.Context, cmd *cobra.Command) {
+	if cmd.Name() == "start" {
+		serverCtx := server.GetServerContextFromCmd(cmd)
+		cmdCtx := cmd.Context()
```

Comment:
```suggestion
		ctx := cmd.Context()
```

we can name context.Context as `ctx`